cloud/auth/auth.go

```diff
@@ -373,20 +372,20 @@ func checkToken(c *config.Context, client astro.Client, out io.Writer) error {
 }
 
 // Login handles authentication to astronomer api and registry
-func Login(domain, orgID string, client astro.Client, out io.Writer, shouldDisplayLoginLink, shouldLoginWithToken bool) error {
+func Login(domain, orgID, shouldLoginWithToken string, client astro.Client, out io.Writer, shouldDisplayLoginLink bool) error {
```

Comment:
```suggestion
func Login(domain, orgID, token string, client astro.Client, out io.Writer, shouldDisplayLoginLink bool) error {
```

since it is a token string, it is better to not name it as `shouldLoginWithToken` which is more of a boolean variable naming convention.

Reply:
Agreed. Was trying to re-use the same variable, but i can change that to a proper string convention